src/Microsoft.Azure.WebJobs.StorageProvider.Blobs/AzureBlobBuilderExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Azure.WebJobs.StorageProvider.Blobs
+{
+    /// <summary>
+    /// Extension methods for Storage Blobs integration.
+    /// </summary>
+    public static class AzureBlobBuilderExtensions
+    {
+        /// <summary>
+        /// Adds the Storage Blobs extension to the provided <see cref="IServiceCollection"/>.
```

Comment:
Is this description accurate? This adds the host services. Extensions, in the context of WebJobs, are something different.